target/linux/ath79/image/generic.mk

```diff
@@ -1544,6 +1544,15 @@ define Device/openmesh_om2p-v4
 endef
 TARGET_DEVICES += openmesh_om2p-v4
 
+define Device/openmesh_om2p-hs-v4
+  $(Device/openmesh_om2p-common)
+  SOC := qca9533
+  DEVICE_MODEL := OM2P-HS
```

Comment:
Since we overwrite it here anyway, I'd remove the DEVICE_MODEL from the common definition and put it directly into openmesh_om2p-v4 as well.

Reply:
only 5 of 8 of devices overwrite it. but ok, it is definitely more consistent to have it always in the actual device definition.

consider this done